sound/soc/sof/ipc4-topology.c

```diff
@@ -1480,6 +1480,10 @@ static int sof_ipc4_widget_setup(struct snd_sof_dev *sdev, struct snd_sof_widget
 		struct snd_sof_dai *dai = swidget->private;
 		struct sof_ipc4_copier *ipc4_copier = dai->private;
 
+		/* set the pipeline that the DAI widget belongs to as a backend pipeline */
+		pipeline = pipe_widget->private;
+		pipeline->is_backend = true;
```

Comment:
Anticipating a 'yes, that's correct' answer:
I guess it is forbidden to have a pipeline directly connecting AIF to DAI, we must always have at least two pipelines running between AIF and DAI?

Reply:
Yes, thats correct

Reply:
we will not be using the pipeline trigger IPC for chained DMA. so it doesnt matter